roles/apache/tasks/main.yml

```diff
@@ -52,10 +52,9 @@
     - restart apache2
 
 
-## RIGHT NOW THIS WILL ONLY WORK FOR API!
 - name: write /etc/apache2/opentree-shared.conf
   template:
-    src: templates/opentree_devapi_conf.j2
+    src: "{{ 'templates/opentree_devapi_conf.j2' if 'phylesystemapi' in role_names else 'templates/opentree_shared_conf.j2' }}" 
```

Comment:
@snacktavish This test of `... in role_names` seems to work well (see also debug output in subsequent tasks).